doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1444 @@
+    The tool allows you to create and edit Matter Manufacturer Cluster files or create an extension to the existing one.
+    The tool is available in release artifacts.
+
+* Updated:
+
+  * By disabling the :ref:`mpsl` before performing a factory reset to speed up the process.
+  * The :ref:`ug_matter_device_low_power_configuration` page to mention the `nWP049 - Matter over Thread: Power consumption and battery life`_ and `Online Power Profiler for Matter over Thread`_ as useful resources in optimizing the power consumption of a Matter device.
+  * The general documentation on trusted storage by moving it to the :ref:`trusted_storage_in_ncs` page and :ref:`trusted_storage_readme` library documentation.
+
+Matter fork
++++++++++++
+
+The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, the ``5fd234d4f14e1225533eaea85854f160bbd0fd55`` commit from the ``v1.4-branch``.
+The following list summarizes the most important changes inherited from the upstream Matter:
+
+* Added:
+
+  * Enhanced Setup Flow that allows the standard Matter commissioning process to enable display and acknowledgment of device makers' legal terms and conditions before the device setup.
+  * Onboarding Payload in NFC tags.
+  * Large messages over TCP.
+  * New ``kFactoryReset`` event that is posted during a factory reset.
+    The application can register a handler and perform additional cleanup
+
+Thread
+------
+
+* Added:
+
+  * Support for storing the Thread key materials in the :ref:`ug_ot_thread_security_kmu`.
+  * The :ref:`ug_ot_thread_security` user guide describing the security features of the OpenThread implementation in the |NCS|.
+
+Zigbee
+------
+
+* Removed all Zigbee resources.
+  They are now available as separate `Zigbee R22`_ and `Zigbee R23`_ add-on repositories.
+
+Wi-Fi
+-----
+
+* Updated:
+
+  * Throughputs for Wi-Fi usage profiles.
+  * The Wi-Fi credential shell, by renaming it from ``wifi_cred`` to ``wifi cred``.
+  * The :ref:`ug_wifi_regulatory_certification` documentation by moving it to the :ref:`ug_wifi` protocol page.
+
+Applications
+============
+
+* Added the new :ref:`hpf_mspi_example` application.
+* Removed the Asset Tracker v2 application.
+  For the development of asset tracking applications, refer to the `Asset Tracker Template <Asset Tracker Template_>`_.
+
+  The factory-programmed Asset Tracker v2 firmware is still available to program the nRF91 Series devices using the `Programmer app`_, `Quick Start app`_, and the `Cellular Monitor app`_.
+
+* Renamed the SDP GPIO application to :ref:`hpf_gpio_example`.
+
+Machine learning
+----------------
+
+* Updated:
+
+  * The application to enable the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` file system for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+  * The Edge Impulse URI configuration to use the new model location.
+
+IPC radio firmware
+------------------
```

Comment:
If we stick to alphabetical order of the subsections - please move _IPC radio firmware_ above _Machine learning_.

Reply:
Moved.